src/utils/maxLiquidityForAmounts.ts

```diff
@@ -2,6 +2,13 @@ import { BigintIsh } from '@uniswap/sdk-core'
 import JSBI from 'jsbi'
 import { Q96 } from '../internalConstants'
 
+/**
+ * Returns an imprecise maximum amount of liquidity received for a given amount of token 0
```

Comment:
Would be great to explain what makes this imprecise (I don't know myself and expect other devs wouldn't know either)